recipe/meta.yaml

```diff
@@ -100,8 +100,14 @@ outputs:
         - numba >=0.58.1
         - attrs >=22.2
         # Keep this in sync with TileDB-SOMA's somacore version requirement.
-        - somacore ==1.0.26
+        - somacore ==1.0.28
         - scanpy >=1.9.2
+        # See TileDB-SOMA's apis/python/requirements_spatial.txt
```

Comment:
I thought we were saving the spatial experiment until a later date

Reply:
Yup, copy/paste error on this draft PR -- fixing after meetings -- thanks for the catch!